java/examples/src/main/java/glide/examples/ExamplesApp.java

```diff
@@ -0,0 +1,60 @@
+/** Copyright GLIDE-for-Redis Project Contributors - SPDX Identifier: Apache-2.0 */
+package glide.examples;
+
+import glide.api.RedisClient;
+import glide.examples.clients.GlideClient;
+import java.util.concurrent.ExecutionException;
+
+public class ExamplesApp {
+
+    // main application entrypoint
+    public static void main(String[] args) {
+        runGlideExamples();
+    }
+
+    private static void runGlideExamples() {
+        ConnectionSettings settings = new ConnectionSettings("localhost", 6379, false, false);
+
+        try {
+            RedisClient client = GlideClient.connectToGlide(settings);
+
+            System.out.println("Glide PING: " + client.customCommand(new String[] {"ping"}).get());
```

Comment:
Consider update this for varargs in custom command

Reply:
We can update this later once the get/set/ping/info PR is merged